tests/integration/tier0/non-destructive/cve-2022-1662/test_cve-2022-1662.py

```diff
@@ -12,8 +12,11 @@ def watchdog():
         for process in psutil.process_iter():
             # For some reason the psutil catches subscription-manager in process.name()
             # as 'subscription-ma', thus using 'subscription' to catch it
-            if "subscription" in process.name():
-                return process.cmdline()
+            try:
+                if "subscription" in process.name():
+                    return process.cmdline()
+            except psutil.NoSuchProcess:
```

Comment:
why this? Please leave a comment saying why we're catching this particular exception. (I do not have any problem with having it here

Reply:
Right, I will add the comment and merge this, since the tests are passing.

Reply:
<img src="https://media3.giphy.com/media/3oEdva9BUHPIs2SkGk/giphy.gif"/>